app/src/main/java/com/infomaniak/drive/views/FileInfoActionsView.kt

```diff
@@ -454,52 +447,75 @@ class FileInfoActionsView @JvmOverloads constructor(
 
     interface OnItemClickListener {
 
-        val ownerFragment: Fragment
-        val currentFile: File
+        val ownerFragment: Fragment?
+        val activity: Activity?
+        val currentFile: File?
 
-        private val context: Context get() = ownerFragment.requireContext()
+        private val context: Context get() = ownerFragment?.requireContext() ?: activity!!
 
         private fun trackFileActionEvent(name: String, value: Boolean? = null) {
             context.trackFileActionEvent(name, value = value?.toFloat())
         }
 
+        fun shareFile() = trackFileActionEvent("shareFile")
+        fun saveToKDriveClicked() = trackFileActionEvent("saveToKDrive")
+        fun printClicked() = trackFileActionEvent("print")
```

Comment:
Is "print" clear enough?

Reply:
printPdf? Since we can only print PDF for the moment.